doc/document_format/document_format_v1.md

```diff
@@ -0,0 +1,96 @@
+# transcribee v1 document Format Version 1
+
+This document describes the first version of internal document format transcribee uses (named "transcribee v1 format" or "transcribee format" in the following text).
+The format is an extension of the [transcribee format](./index.md).
+
+This document contains a description of the content of a document in the transcribee v1 format (named "transcribee v1 document" in the following text).
+It also contains restrictions / rules of the content.
+
+The information in this document are sufficient to read / write a transcribee v1 document.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 [RFC2119] [RFC8174](https://datatracker.ietf.org/doc/html/rfc8174) when, and only when, they appear in all capitals, as shown here.
+
+## General
+
+A transcribee v1 document consists of the following things:
+
+1. paragraphs
+2. speaker names
+3. diarization data
+
+## Paragraphs
+
+If a transcribee v1 document contains paragraphs, they MUST be stored in the `children` property of the transcribee v1 document as an array.
+Each array element MUST have the following properties:
+
+- `lang`: The `lang` MUST be a string or null. If not null, it MUST be an ietf language code. It MAY NOT be another value.
+- `speaker`: The id of the speaker of the current paragrpah. It MUST be an integer of null.
+- `alternative_speakers`: A list of other possible speakers for this paragraph. The `alternative_speakers` MUST be an array of integers. It MAY be empty. It MUST be sorted in ascending order.
+- `children`: The `children` MUST be an array of _text atoms_.
+- `type`: MUST be set to the string `paragraph`
+
+If the `speaker` is generated automatically, it SHOULD only be set if there is only one possible speaker for this paragraph.
```

Comment:
Do we have a speaker confidence? Then we could handle this like text and automatically select the best guess and add a mark if the confidence is low.

Reply:
Not really with the current models I think.